skins/oasis/js/privacyUpdateNotification.js

```diff
@@ -4,8 +4,30 @@ $(function() {
         // display notification
         var notification = new BannerNotification(message).show();
 
+        function getCookieDomain(hostname) {
+            const parts = hostname.split('.');
+            if (parts.length < 2) {
+                return undefined;
+            }
+        
+            let cookieDomain = `.${parts[parts.length - 2]}.${parts[parts.length - 1]}`;
+            // These exceptions require a third part for a valid cookie domain. This isn't
+            // a definitive list but rather the most likely domains on which Fandom would
+            // host a site.
+            const exceptions = [
+                '.co.jp',
+                '.co.nz',
+                '.co.uk',
+            ];
+            if (exceptions.indexOf(cookieDomain) >= 0) {
+                cookieDomain = `.${parts[parts.length - 3]}${cookieDomain}`;
```

Comment:
Is ``` template literals supported on app?

Reply:
They shouldn't be, ResourceLoader does not accept accept code that is syntactically invalid in the ES3 standard.

Reply:
Just caught that as well.